backend/app/deps/authorization_deps.py

```diff
@@ -322,7 +322,7 @@ async def __call__(
                 return True
             for u in group.users:
                 if u.user.email == current_user:
-                    if u.editor and self.role == RoleType.EDITOR:
+                    if u.editor or self.role == RoleType.EDITOR:
```

Comment:
Unless I'm misunderstanding, I don't think this should be changed. `self.role` is the role you are requiring in the endpoint, so this would mean even if the user `u` is not an editor, if your permission requires editor this would return true for any user in the group.

Reply:
ah i see i misunderstood the `self.role` part. Sorry about that. 
Will revert it